src/components/Team.jsx

```diff
@@ -17,6 +17,22 @@ const Team = () => {
           <Member name={name} position={position} image={image} key={index} />
         ))}
       </div>
+      <div className="text-2xl lg:text-3xl font-header my-4">Committees</div>
+      <div className="grid grid-cols-1 md:grid-cols-2">
```

Comment:
maybe add a small gap like `gap-2` or so for better separation in `grid-cols-1` for small displays?

![image](https://github.com/bearhack-ucr/bearhack2024/assets/12877445/18ed00ca-a380-4d47-8d1a-cabf7d421795)